cmd/keytransparency-client/cmd/hammer.go

```diff
@@ -68,13 +68,13 @@ var hammerCmd = &cobra.Command{
 		if err != nil {
 			log.Fatal(err)
 		}
-		handle, err := tink.NewKeysetHandleFromReader(
+		handle, err := keyset.Read(
 			&tinkio.ProtoKeysetFile{File: keysetFile},
 			masterKey)
 		if err != nil {
 			log.Fatal(err)
 		}
-		keyset = handle
+		ks = handle
```

Comment:
This is a bit confusing. Can you directly use handle, without reassigning to ks?

Reply:
yeah, I can move this to the `RunE` function